pkg/kubelet/cadvisor/util_test.go

```diff
@@ -55,3 +56,44 @@ func TestCapacityFromMachineInfoWithHugePagesEnable(t *testing.T) {
 func TestCrioSocket(t *testing.T) {
 	assert.EqualValues(t, CrioSocket, crio.CrioSocket, "CrioSocket in this package must equal the one in github.com/google/cadvisor/container/crio/client.go")
 }
+
+func TestDefaultCadvisorMetricSet(t *testing.T) {
+	usingLegacyStatsTable := []bool{false, true}
+	metricSet := cadvisormetrics.MetricSet{}
+	for metricKind := range IncludedMetrics {
+		metricSet.Add(metricKind)
+	}
+	metricSet.Add(cadvisormetrics.DiskUsageMetrics)
+
+	expected := []cadvisormetrics.MetricSet{IncludedMetrics, metricSet}
+
+	for idx, usingLegacyStats := range usingLegacyStatsTable {
+		actual := DefaultCadvisorMetricSet(usingLegacyStats)
+		if reflect.DeepEqual(actual, expected[idx]) {
+			t.Errorf("test default metric set for cadvisor error, expected: %+v, actual: %+v", expected[idx], actual)
+		}
+	}
+}
+
+func TestCustomCadvisorMetricSet(t *testing.T) {
+	usingLegacyStatsTable := []bool{false, true}
```

Comment:
does this test cover the check on `kubefeatures.DisableAcceleratorUsageMetrics`?